graphrag_sdk/steps/create_ontology_step.py

```
@@ -38,21 +37,48 @@
         sources: list[AbstractSource],
         ontology: Ontology,
         model: GenerativeModel,
-        config: dict = {
+        config: Optional[dict] = {
             "max_workers": 16,
             "max_input_tokens": 500000,
             "max_output_tokens": 8192,
         },
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)